libs/wire-api/src/Wire/API/Asset.hs

```diff
@@ -200,9 +200,10 @@ newtype AssetToken = AssetToken {assetTokenAscii :: AsciiBase64Url}
 
 instance ToSchema AssetToken where
   schema =
-    AssetToken <$> assetTokenAscii
-      .= schema
-        & doc' . S.schema . S.example ?~ toJSON ("aGVsbG8" :: Text)
+    AssetToken
+      <$> assetTokenAscii
+        .= schema
+      & doc' . S.schema . S.example ?~ toJSON ("aGVsbG8" :: Text)
```

Comment:
Is this right? I am actually not sure if `doc'` is applied to `schema` or `(AssetToken <$> assetTokenAcii .= schema)`. From the original code it looks like it is applied to `schema`, now it looks like that later.

Reply:
`&` is `infixl 1` so it is applying to `AssetToken <$> assetTokenAscii .= schema`